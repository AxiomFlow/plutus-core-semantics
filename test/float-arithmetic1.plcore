(builtin subtractFloat
  ((builtin multiplyFloat (2.0 (builtin multiplyFloat (3.0 10.0)))) 5.0))
