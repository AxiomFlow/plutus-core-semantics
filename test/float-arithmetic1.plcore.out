<k> 5.5e+01 </k>
